crates/bevy_pbr/src/light.rs

```diff
@@ -28,7 +28,7 @@ impl Default for PointLight {
 pub(crate) struct PointLightUniform {
     pub pos: [f32; 4],
     pub color: [f32; 4],
-    pub inverse_range_squared: f32,
+    pub inverse_range_squared: [f32; 4],
```

Comment:
Can we add a note here indicating why this is an array?

Reply:
done also for `LightCount.num_lights` while here